### PR TITLE
Use k8s.io/component-base/cli to run cmd

### PR DESCRIPTION
**Pull request by wuyingjun-lucky**
Signed-off-by: wuyingjun <wuyingjun_yewu@cmss.chinamobile.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
refer to [kubernetes api server ](https://github.com/kubernetes/kubernetes/blob/f92152da72bf2de312a54c77ef234e94dabf61c8/cmd/kube-apiserver/apiserver.go#L32-L34)
kubernetes replace all  commands with k8s.io/component-base/cli to run
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/agent/main.go [hunk 1]
```diff
@@ -15,25 +15,15 @@ import (
 	_ "sigs.k8s.io/controller-runtime/pkg/metrics"
 
 	apiserver "k8s.io/apiserver/pkg/server"
-	"k8s.io/component-base/logs"
+	"k8s.io/component-base/cli"
+	_ "k8s.io/component-base/logs/json/register"
```
Comment (XiShanYongYe-Chang) [inline cmd/agent/main.go:19]: Any comment for this import? Just like:
```
// for JSON log format registration
```

**Review by XiShanYongYe-Chang — COMMENTED** [review]
@wuyingjun-lucky Thanks~

Just a question: is the log format changed before and after the modification?

File: cmd/agent/main.go (hunk 1 above)
Comment (wuyingjun-lucky) [reply cmd/agent/main.go]: > Any comment for this import? Just like:
> 
> ```
> // for JSON log format registration
> ```

DONE

**Comment by lonelyCZ** [comment]
Looks great. Thanks @wuyingjun-lucky 

/lgtm

**Pull request merged**